### DISCUSSION TITLE
eos在以太链未做映射，如何进行映射找回

### DISCUSSION THREAD

**Discussion by ivu100** [famland/website#56] [General]
eos在以太链在2018年未做映射，未生成私钥，现在如何把以太链的EOS进行映射到EOS链

**Comment by otto-001** [comment]
你这个问题，太久远了。 建议到eos基金会社区群去问问
eos社区中文电报群：https://t.me/EOSCN

EOS网络基金会Discord 
https://discord.com/invite/eos-network
EOS网络基金会中文Twitter
https://twitter.com/EOSFoundationCN
EOS网络基金会中文YouTube
https://www.youtube.com/channel/UCCoFKMELHUVQR2DjAcRXp6w
